packages/shared/src/components/comments/ScoutBadge.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { ReactElement } from 'react';
+import ScoutIcon from '../../../icons/filled/scout.svg';
+
+export default function ScoutBadge(): ReactElement {
+  return (
+    <span className="flex items-center ml-2 text-theme-color-bun typo-footnote">
+      <ScoutIcon className="mr-0.5 text-base icon" />
```

Comment:
I think it should be `mr-1`.
![image](https://user-images.githubusercontent.com/13744167/168226805-4b5691ed-5a5d-419e-b242-bfa7ce51fb0c.png)

I don't think you should be using font size for the image size. Also, the `icon` class shouldn't be necessary. I don't think we have a specific selector in our base CSS for this.
```suggestion
      <ScoutIcon className="mr-1 w-3 h-3" />
```